There are more than one st_{a,m,c}tim fields in struct stat on time64 machines.

Run the Go-isation on all of them.

--- gcc-8.3.0/libgo/mksysinfo.sh.old	2018-11-09 15:29:52.000000000 +0000
+++ gcc-8.3.0/libgo/mksysinfo.sh	2019-12-23 00:18:54.129911814 +0000
@@ -468,9 +468,9 @@
          -e 's/st_size/Size/' \
          -e 's/st_blksize/Blksize/' \
          -e 's/st_blocks/Blocks/' \
-         -e 's/st_atim/Atim/' \
-         -e 's/st_mtim/Mtim/' \
-         -e 's/st_ctim/Ctim/' \
+         -e 's/st_atim/Atim/g' \
+         -e 's/st_mtim/Mtim/g' \
+         -e 's/st_ctim/Ctim/g' \
          -e 's/\([^a-zA-Z0-9_]\)_timeval\([^a-zA-Z0-9_]\)/\1Timeval\2/g' \
          -e 's/\([^a-zA-Z0-9_]\)_timespec_t\([^a-zA-Z0-9_]\)/\1Timespec\2/g' \
          -e 's/\([^a-zA-Z0-9_]\)_st_timespec_t\([^a-zA-Z0-9_]\)/\1StTimespec\2/g' \
